src/compose.ts

```diff
@@ -493,7 +493,6 @@ export interface ComposeFactory<T, O extends Options> extends ComposeMixinable<T
 	/**
 	 * Extend the factory prototype with the supplied object literal, class, or factory
 	 *
-	 * @deprecated
```

Comment:
I would think so. Unless I am misunderstanding, we deprecated `extend` because we wanted to discourage the pattern of adding new properties to a compose class. But we still do that all over the place with `mixin`(or at least I am in stores). 

These changes would leave the old `mixin` api for backwards compatibility but it would really be discouraged, so I think we would need a different way to add properties to a prototype.